services/ui-src/src/components/reports/ReportPageIntro.tsx

```diff
@@ -27,13 +26,18 @@ export const ReportPageIntro = ({
         {initiativeName ? initiativeName : subsection}
       </Heading>
       {hint && <Box sx={sx.hintTextBox}>{hint}</Box>}
-      {accordion && <InstructionsAccordion verbiage={accordion} />}
-      {stepType === "evaluationPlan" && report?.reportType === ReportType.SAR && (
+      {accordion && stepType !== "objectiveProgress" && (
+        <InstructionsAccordion verbiage={accordion} />
+      )}
+      {stepType === "objectiveProgress" && (
         <Heading as="h3" sx={sx.subTitle}>
           Objectives progress
         </Heading>
       )}
```

Comment:
Side note: This isn't part of the ticket but it stuck out to me. Feels like this could go into the page's info and not need to be added here like this.. Good call on getting rid of the useState call! I also think thats probably no longer necessary.

Reply:
i updated it to pull dynamically from the text prop instead of being hardcoded - but feel free to create a bug tkt if you'd like to address it further!